packages/wrangler/src/api/dev.ts

```diff
@@ -0,0 +1,32 @@
+import { startDev } from "../dev";
+import { logger } from "../logger";
+interface DevOptions {
+	env?: string;
+	ip?: string;
+	port?: number;
+	localProtocol?: "http" | "https";
+	assets?: string;
+	site?: string;
+	siteInclude?: string[];
+	siteExclude?: string[];
+	nodeCompat?: boolean;
+	experimentalEnableLocalPersistence?: boolean;
+	_: (string | number)[]; //yargs wants this
+	$0: string; //yargs wants this
+}
```

Comment:
Why are only a subset of the `ArgumentsCamelCase<DevArgs>` properties used here?

I think we should make `DevOptions` something inside `dev.tsx`, then import it here.
This would avoid us having to mention yargs here.

I think we should also export the type of the options that is expected by the public API `unstable_dev` so that clients can use that type to build up their options correctly. (Otherwise they would have to resort to something like: `const options: Arguments<typeof unstable_dev>[1]`.

Reply:
I wanted to narrow the options passed into the API (to start with anyway)